user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md

```diff
@@ -55,11 +55,28 @@ This file will contain all necessary information to register a Catalog item or v
 
 ```yml
 # [Required]
+# A short description of the types can be found here: [Catalog types](xref:About_the_Catalog_module#supported-catalog-item-types).
```

Comment:
Within a code block, a link in this format will not work. It would be better to add an aka link to the page here.